### PR TITLE
merge file structure with overview section

### PR DESCRIPTION
**Pull request by natashamm**
From https://github.com/collective/icalendar/issues/626

I didn't change much of the content for now.

----
📚 Documentation preview 📚: https://icalendar--739.org.readthedocs.build/

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
 - :rfc:`9073` - Event Publishing Extensions to iCalendar
 - :rfc:`9253` - Support for iCalendar Relationships
 
-File structure
+iCalendar File structure
 --------------
```
Comment (stevepiercy) [inline docs/usage.rst]: In reStructuredText, the underline length must match that of its heading.

```suggestion
------------------------
```

File: docs/usage.rst [hunk 2]
```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
...
 
-An iCalendar file is a text file (utf-8) with a special format. Basically it
-consists of content lines.
+An iCalendar file is a text file (utf-8) with a special format.
```
Comment (stevepiercy) [inline docs/usage.rst]:
```suggestion
An iCalendar file is a text file with UTF-8 character encoding in a special format.
```

File: docs/usage.rst [hunk 3]
```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
...
 
-Each content line defines a property that has 3 parts (name, parameters,
-values). Parameters are optional.
+It consists of **content lines**,
+with each content line defining a property that has 3 parts: name, parameters and value. Parameters are optional.
```
Comment (stevepiercy) [inline docs/usage.rst]:
```suggestion
with each content line defining a property that has 3 parts: name, parameters, and values. Parameters are optional.
```

File: docs/usage.rst [hunk 4]
```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
...
 
-A simple content line with only name and value could look like this::
+Example 1: a simple content line (with only name and value)::
```
Comment (stevepiercy) [inline docs/usage.rst]:
```suggestion
Example 1: a simple content line, with only name and value.
```

File: docs/usage.rst [hunk 5]
```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
...
 
   BEGIN:VCALENDAR
```
Comment (stevepiercy) [inline docs/usage.rst]: For each example, the default lexer is Python by virtue of the preceding line's `::`, but should be text. Would you please update each example with this pattern, replacing `::` with `.`, inserting the `code-block` directive, and indenting by four spaces the code example?

```suggestion
.. code-block:: text

    BEGIN:VCALENDAR
```

File: docs/usage.rst [hunk 6]
```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
...
-And the parts are::
+The parts in this example are::
 
   Name:   ATTENDEE
   Params: CN=Max Rasmussen;ROLE=REQ-PARTICIPANT
   Value:  MAILTO:example@example.com
 
-Long content lines are usually "folded" to less than 75 character, but the
-package takes care of that.
+Note: Long content lines are usually "folded" to less than 75 characters (the
```
Comment (stevepiercy) [inline docs/usage.rst]: Change to active from passive voice.

```suggestion
For long content lines, iCalendar usually "folds" them to less than 75 characters.
```

File: docs/usage.rst [hunk 7]
```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
...
+package takes care of this).
```
Comment (stevepiercy) [inline docs/usage.rst]:
```suggestion
```

File: docs/usage.rst [hunk 8]
```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
...
 
+On a higher level, you can think of iCalendar files structured as having components and sub components.
```
Comment (stevepiercy) [inline docs/usage.rst]:
```suggestion
On a higher level, you can think of iCalendar files' structure as having components and subcomponents.
```

File: docs/usage.rst [hunk 9]
```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
...
 
-Overview
---------
+A component will have properties with values. The values
+have special types, like integer, text, and datetime. These values are
+encoded in a special text format in an iCalendar file. This package contains methods for converting to and from these encodings.
```
Comment (stevepiercy) [inline docs/usage.rst:62]: I've never been a fan of arbitrary line wrapping in narrative documentation, but if we're going to do that, then we should be consistent.

Narrative documentation is not code, and we're no longer constrained by 80-character terminal sessions over dial-up modems when editing. To me, it makes more sense to have one sentence per line, eliminating the need for line wrapping and making for easier review of diffs, but I won't push for it.

File: docs/usage.rst [hunk 10]
```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
...
-Inside the components there are properties with values. The values
-have special types. Like integer, text, datetime etc. these values are
-encoded in a special text format in an iCalendar file.
 
-There are methods for converting to and from these encodings in the package.
+Components
+----------
 
-These are the most important imports::
+Note: The code snippets remainder of the docs will use these important imports::
```
Comment (stevepiercy) [inline docs/usage.rst]: `pycon` is the Pygments Python console lexer. See https://pygments.org/languages/.

```suggestion
The remaining code snippets in the documentation will use the following important imports.

.. code-block:: pycon
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Some syntax improvements, grammar fixes. Please spell out words, as it makes it easier to translate them.

**Comment by niccokunzmann** [comment]
I think, the docs pass the build. @natashamm , could you add a changelog entry? Let me know when you are done improving the docs! Thanks :)

**Comment by niccokunzmann** [comment]
Also, you can look here to see if it is how you like it: https://icalendar--739.org.readthedocs.build/en/739/usage.html

File: docs/usage.rst (hunk 9 above)
Comment (natashamm) [reply docs/usage.rst:62]: Agreed either way it should be consistent. Are you OK with creating a separate issue to discuss the line wrapping in the docs?

File: docs/usage.rst (hunk 5 above)
Comment (natashamm) [reply docs/usage.rst]: Done!

**Comment by natashamm** [comment]
@niccokunzmann it is ready to go if the line wrapping conversation can be resolved!

File: docs/usage.rst (hunk 9 above)
Comment (stevepiercy) [reply docs/usage.rst:62]: Content is good.

I won't enforce
line wrapping, nor do I want to
spend time on it. But it's something to consider
for future PRs.

😜

**Comment by niccokunzmann** [comment]
Thanks @natashamm and @stevepiercy!

**Pull request merged**